rfcs/0043-rfcsc-rotation.md

```diff
@@ -0,0 +1,102 @@
+that RFC included a definition of how members to the Committee are to be chosen
+or removed. This RFC provides mechanisms through which membership in the
+Steering Committee can be established or ended, so as not to make membership a
+mandate for life, allowing a healthy rotation of members, and offering several
+procedures to always keep the Steering Committee capable of making decisions
+such that the RFC process can keep running smoothly at all times.
+
+# Detailed design
+[design]: #detailed-design
+
+The RFC Steering Committee shall always have five members. If membership drops
+below five members (for example by resignation of a member as detailed below), a
+new member shall directly be elected after a nomination period of at least two
+weeks (see below for nomination and selection process). If the number of
+members of the RFCSC drops below 4 people, it cannot proceed with shepherd team
+selections until new members have been selected.
+
+## Ending membership
+A member can end their membership in the Steering
+Committee by either of the following four mechanisms, ordered from most
+frequent to least:
+
+1. At the end of an election period.
+2. Resignation
+3. Unanimous vote by all other members after having missed two or more regular
+   meetings without giving an appropriate excuse.
+4. Unanimous vote by all other members due to conduct unbecoming of a member.
+
+A member can resign from the RFC Steering Committee at any time and for any
+reason. A member planning to resign should inform the rest of the RFC
+Steering Committee of their intention at their earliest convenience.
+
+## Becoming a member
+The members chosen through the original implementation as
+specified in RFC 36 are regular members as specified in this RFC. They will
+stay on as members until replaced by new members as outlined below.
+
+If a seat has to be filled earlier than at the yearly vote, the new member will
+only serve for the rest of the term.
+
+Each year at the first meeting of the RFCSC in December (starting 2019,
+approximately a year after establishment in RFC 36) they unanimously decide on
+the succeeding committee members. If unanimous agreement cannot be reached, the
+RFCSC votes on each nominee, ranking them and further hold run-off votes if
+there is a tie. Nominations are open to anyone, and one can either nominate
+themself or any other person who accepts the nomination. Members of the
+previous RFCSC explicitly can stand again, but should reflect on their free
+time and commitment to their role. The nomination period starts at the
+beginning of November, a minimum of four weeks before election, and is
+announced on all relevant communication channels (as of April 2019,
+discourse.nixos.org, IRC #nixos and #nixos-dev, and NixOS Weekly).
+
+Additionally to RFC 36 a new restriction formally comes into effect. In order
+to avoid conflict of interest there is an upper cap of 2 members working for a
+single employer.
```

Comment:
1. Does that mean that if a member changes employment in a cap-violating way, they are expected to resign (unless one of the two members from the new employment place prefers to resign themselves)? Should this be explicitly recorded? 

2. I want to have this question on the record: do we want to respect the other dimensions of conflict of interest? I think it would be feasible to ask that if a full SC of five members is being selected, at least one member of the incoming committee should have used Nix outside the single most popular setup (currently NixOS on `x86_64`). It would be a nightmare in a large election procedure, but for unanimous selection of successors it seems feasible.

Reply:
1. I don't think it is necessary, but if someone cares to resign no one will stop them.
2. I also think we don't have to specify this, as the RFCSC should always try and set up a balanced shepherding team, familiar with the affected parts and the RFCSC can realistically not cover everything themself.

Reply:
1. I am OK with that, but then maybe reword it to say that there is a cap of _appointing_ at most 2 people working for the same employer? Because right now it seems to mean my interpretation.

2. It might be easier to evaluate experience (and relevance of that experience) of people working on special use cases for a SC including someone who doesn't use the «default setup» exclusively. But I do not insist.